pkg/services/srvctest/health.go

```diff
@@ -0,0 +1,17 @@
+package srvctest
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"golang.org/x/exp/maps"
+	"golang.org/x/exp/slices"
+)
+
+func AssertHealthReportNames(t *testing.T, hp map[string]error, names ...string) {
```

Comment:
What are some other ways we will want to use this kind of API?
I imagine we will often want to verify they are all nil, or that some in particular are non-nil or match certain errors, but those things may be better done as a separate funcs :thinking:

Reply:
This might be too simple to be worth it:
```go
func AssertHealthy(t *testing.T, hp map[string]error) {
	t.Helper()
	require.NoError(t, errors.Join(maps.Values(hp)...))
}
```